target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml

```diff
@@ -4,4 +4,18 @@
   <artifactId>flipkart.web.automation</artifactId>
```

Comment:
Remove this file too. only by adding target folder in .gitignore would not include this file next time.

Reply:
Lets make it a practice to use versions that will not change so frequently in properties tag , like this.
```
   <name>Flipkart Web Automation</name>
<properties>
		<maven.compiler.source>1.8</maven.compiler.source>
		<maven.compiler.target>1.8</maven.compiler.target>
</properties>
```

and then use in plugin like this

```
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-compiler-plugin</artifactId>
<version>3.7.0</version>
<configuration>
<source>${maven.compiler.source}</source>
<target>${maven.compiler.target}</target>
</configuration>
```